docs/source/snapshot.rst

```diff
@@ -186,15 +195,17 @@ are supported:
 .. note::
 
   During execution of
-  :py:meth:`~capella_rm_bridge.change_set.change.TrackerChange.calculate_change` the
-  integrity of the snapshot is checked. That means for example work items that
-  have ``type`` identifiers which are not defined in the
+  :py:meth:`~capella_rm_bridge.change_set.change.TrackerChange.calculate_change`
+  the integrity of the snapshot is checked. That means for example work items
+  that have ``type`` identifiers which are not defined in the
   :ref:`requirement_types` section will be skipped. In general there needs to
   be a ``type`` identifier exported in order to have fields maintained.
 
   Another example: If there are any options/values exported on an enum-field
   which are not defined in the respective enum definition under ``data_types``,
-  the field will be skipped.
+  the field will be skipped if the force mode is enabled. If force mode is
+  disabled (default) any error will result into cancelation of the ChangeSet
+  calculation.
 
 With the ``children`` key the hierarchical structure of the workitems is
 exported and empty children will result in a ``Requirement``. Conversely
```

Comment:
This paragraph needs to be adjusted to the new behavior described further up (existence of the key is important, not its value).